metadata-io/src/main/java/com/linkedin/metadata/entity/EntityService.java

```diff
@@ -131,11 +125,8 @@ public abstract RecordTemplate getAspect(
    * @param auditStamp an {@link AuditStamp} containing metadata about the writer & current time
    * @return the {@link RecordTemplate} representation of the written aspect object
    */
-  public abstract RecordTemplate ingestAspect(
-      @Nonnull final Urn urn,
-      @Nonnull final String aspectName,
-      @Nonnull final RecordTemplate newValue,
-      @Nonnull final AuditStamp auditStamp);
+  public abstract RecordTemplate ingestAspect(@Nonnull final Urn urn, @Nonnull final String aspectName,
```

Comment:
Why these changes? Is this checkstyle? I much prefer new lines for readability.

Reply:
Tweaked the linkedin formatter to make it do new lines for this class. But a lot of other classes follow this style tho.